CHANGES.md

```diff
@@ -1,3 +1,9 @@
+1.8.4
+=====
+
+* [#52](https://github.com/serokell/log-warper/issues/52):
+  Add `System.Wlog.Concurrent` module.
```

Comment:
Maybe tell here what functionality this module contains? Like, it allows to run action in parallel with logging or something like that.